catcher_modules/mq/rabbit.py

```diff
@@ -0,0 +1,111 @@
+from time import sleep
+
+from catcher.steps.check import Operator
+from catcher.steps.external_step import ExternalStep
+from catcher.steps.step import Step, update_variables
+from catcher.utils.file_utils import read_file
+from catcher.utils.logger import debug
+from catcher.utils.misc import try_get_object, fill_template_str
+from catcher.utils.time_utils import to_seconds
+import pika
```

Comment:
please import `pika` locally and add a new module `rabbit` with pika dependency [here](https://github.com/comtihon/catcher_modules/blob/master/setup.py#L25)

Reply:
sure addressed that